tests/baselines/reference/castExpressionParentheses.js

```diff
@@ -74,12 +74,12 @@ a;
 a[0];
 a.b["0"];
 a().x;
+1..foo;
+1..foo;
+1.0.foo;
+12e+34.foo;
+0xff.foo;
 // should keep the parentheses in emit
-(1).foo;
-(1.).foo;
-(1.0).foo;

```

Comment:
Without this change, we over-parenthesize for cases like `0b10.toString()`, which emits `(2).toString()` where we previously would emit `2..toString()` since we were handling this in the emitter.

I think using `..` here is more consistent, and saves a character in the emit.